src/test/java/hudson/plugins/git/browser/GithubWebTest.java

```diff
@@ -8,25 +8,24 @@
 import hudson.plugins.git.GitChangeSet;
 import hudson.plugins.git.GitChangeSet.Path;
 import hudson.plugins.git.GitSCM;
-import hudson.plugins.git.extensions.GitSCMExtension;
 import hudson.scm.RepositoryBrowser;
+import jenkins.plugins.git.AbstractGitSCMSource;
```

Comment:
Please put these imports back in their previous location.

There are many pending pull requests in this repository and each white space change risks causes multiple pull requests to develop conflicts over changes that are purely cosmetic.

The code formatting in the plugin is inconsistent but it is better to maintain that inconsistency than to generate unnecessary conflicts in the many pending pull requests.